source/colors.js

```diff
@@ -15,255 +15,354 @@ module.exports = {
       ]
     },
     {
-      name: 'cool-gray',
+      name: 'cool gray',
       values: [
-        { tone: 'core', value: '#6d7777' },
-        { tone: '10', value: '#dfe9e9' },
-        { tone: '20', value: '#c8d2d2' },
-        { tone: '30', value: '#aeb8b8' },
-        { tone: '40', value: '#959f9f' },
-        { tone: '50', value: '#6d7777' },
-        { tone: '60', value: '#5a6464' },
-        { tone: '70', value: '#3c4646' },
-        { tone: '80', value: '#323c3c' },
-        { tone: '90', value: '#0d1111' },
-        { tone: '100', value: '#000203' }
+        { tone: 'core', value: '#6f7878' },
+        { tone: '1', value: '#e3ecec' },
+        { tone: '10', value: '#d0dada' },
+        { tone: '20', value: '#b8c1c1' },
+        { tone: '30', value: '#9fa7a7' },
+        { tone: '40', value: '#8c9696' },
+        { tone: '50', value: '#6f7878' },
+        { tone: '60', value: '#535a5a' },
+        { tone: '70', value: '#424747' },
+        { tone: '80', value: '#343334' },
+        { tone: '90', value: '#272727' }
       ]
     },
     {
-      name: 'warm-gray',
+      name: 'warm gray',
       values: [
         { tone: 'core', value: '#7d7373' },
-        { tone: '10', value: '#e9e0e0' },
-        { tone: '20', value: '#d0c7c7' },
-        { tone: '30', value: '#b8aeae' },
-        { tone: '40', value: '#9e9494' },
+        { tone: '1', value: '#efe9e9' },
+        { tone: '10', value: '#e2d5d5' },
+        { tone: '20', value: '#ccbcbc' },
+        { tone: '30', value: '#b4a1a1' },
+        { tone: '40', value: '#9e9191' },
         { tone: '50', value: '#7d7373' },
-        { tone: '60', value: '#645a5a' },
-        { tone: '70', value: '#504646' },
-        { tone: '80', value: '#3c3232' },
-        { tone: '90', value: '#1a1314' },
-        { tone: '100', value: '#030000' }
+        { tone: '60', value: '#5f5757' },
+        { tone: '70', value: '#4b4545' },
+        { tone: '80', value: '#373232' },
+        { tone: '90', value: '#2a2626' }
       ]
     },
     {
       name: 'neutral-white',
       values: [
-        { tone: 'core', value: '#fdfdfd' },
-        { tone: '10', value: '#fdfdfd' },
-        { tone: '20', value: '#f9f9f9' },
-        { tone: '30', value: '#f4f4f4' },
-        { tone: '40', value: '#ececec' }
+        { tone: 'core', value: '#fcfcfc' },
+        { tone: '1', value: '#fcfcfc' },
+        { tone: '2', value: '#f9f9f9' },
+        { tone: '3', value: '#f6f6f6' },
+        { tone: '4', value: '#f3f3f3' }
       ]
     },
     {
       name: 'cool-white',
       values: [
         { tone: 'core', value: '#fbfcfc' },
-        { tone: '10', value: '#fbfcfc' },
-        { tone: '20', value: '#f9f9fb' },
-        { tone: '30', value: '#f0f2f4' },
-        { tone: '40', value: '#ecf0f2' }
+        { tone: '1', value: '#fbfcfc' },
+        { tone: '2', value: '#f8fafa' },
+        { tone: '3', value: '#f4f7f7' },
+        { tone: '4', value: '#f0f4f4' }
       ]
     },
     {
       name: 'warm-white',
```

Comment:
De-hyphenate as in https://github.com/IBM-Design/colors/pull/27/files#diff-6a853a1f7639b0af7520dddefa30c2ffR292